Dokan/Dokan/Domain/.gitignore

```diff
@@ -0,0 +1,9 @@
+.DS_Store
```

Comment:
Please remove this auto generated file since we already have .gitignore in the root folder